src/toxcore/binary_io_new.rs

```diff
@@ -46,3 +53,59 @@ impl FromBytes for PublicKey {
 impl FromBytes for Nonce {
     named!(from_bytes<Nonce>, map_opt!(take!(NONCEBYTES), Nonce::from_slice));
 }
+
+// Ip Type
+// Value | Type
+// ----- | ----
+// `2`   | UDP IPv4
```

Comment:
Really? `IpAddr` Does not contain the type.

Reply:
ok, removed this comment - it was present in original code.